File: account/regional-settings/get-states-list/code-samples/listStates.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// OPTIONAL QUERY PARAMETERS
$queryParams = array(
    //'allCountries' => true,
    //'countryId' => 000,
    //'page' => 1,
    //'perPage' => 100,
    //'withPhoneNumbers' => true
);

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->get("/restapi/v1.0/dictionary/state", $queryParams);
// PROCESS RESPONSE
?>

File: account/regional-settings/get-states-list/code-samples/listStates.cs
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// OPTIONAL QUERY PARAMETERS
ListStatesParameters listStatesParameters = new ListStatesParameters {
    //allCountries = true,
    //countryId = 000,
    //page = 1,
    //perPage = 100,
    //withPhoneNumbers = true
};

RestClient rc = new RestClient(
    Environment.GetEnvironmentVariable("clientId"),
    Environment.GetEnvironmentVariable("clientSecret"),
    false
);
await rc.Authorize(
    Environment.GetEnvironmentVariable("username"),
    Environment.GetEnvironmentVariable("extension"),
    Environment.GetEnvironmentVariable("password")
);
var r = await rc.Restapi().Dictionary().State().List(listStatesParameters);
// PROCESS RESPONSE
